static/images/logos/fx-logo.svg

```diff
@@ -0,0 +1,79 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
```

Comment:
Q: Should this be minified?
There doesnt seem to be a specified width/height, so I can't view the SVG in GitHub's Rich Diff tools.